lib/Cleantalk/ApbctWP/FindSpam/UsersChecker.php

```diff
@@ -208,27 +201,29 @@ public static function ctAjaxCheckUsers()
             $from_date = date('Y-m-d', intval(strtotime($_POST['from']))) . ' 00:00:00';
             $till_date = date('Y-m-d', intval(strtotime($_POST['till']))) . ' 23:59:59';
 
-            $from_till = " AND $wpdb->users.user_registered >= '$from_date' AND $wpdb->users.user_registered <= '$till_date'";
+            $from_till = "WHERE $wpdb->users.user_registered >= '$from_date' AND $wpdb->users.user_registered <= '$till_date'";
         }
 
         $wc_orders = '';
 
         if ( $wc_active && ! empty($_POST['accurate_check']) ) {
-            $wc_orders = " AND NOT EXISTS (SELECT posts.* FROM {$wpdb->posts} AS posts INNER JOIN {$wpdb->postmeta} AS postmeta WHERE posts.post_type = 'shop_order' AND posts.post_status = 'wc-completed' AND posts.ID = postmeta.post_id AND postmeta.meta_key = '_customer_user' AND postmeta.meta_value = {$wpdb->users}.ID)";
+        	$sql_command = 'WHERE ';
+        	if ($from_till) {
+		        $sql_command = ' AND ';
+	        }
+            $wc_orders = $sql_command . "NOT EXISTS (SELECT posts.* FROM {$wpdb->posts} AS posts INNER JOIN {$wpdb->postmeta} AS postmeta WHERE posts.post_type = 'shop_order' AND posts.post_status = 'wc-completed' AND posts.ID = postmeta.post_id AND postmeta.meta_key = '_customer_user' AND postmeta.meta_value = {$wpdb->users}.ID)";
```

Comment:
Please, split this in a few rows using concatenation operator.

Reply:
done

Reply:
I don't see any changes in the sql-string request. 

Please, split the long sql-request string in a few rows using concatenation operator. Similar to this one https://github.com/CleanTalk/security-malware-firewall/blob/master/inc/spbc-scanner.php#L487. It's hard to read.